src/xcode/ENA/ENAUITests/ExposureSubmission/ExposureSubmissionUITests.swift

```diff
@@ -353,7 +353,7 @@ class ENAUITests_04_ExposureSubmissionUITests: XCTestCase {
 		type(app, text: "qwdzxcsrhe")
 
 		// Click continue button.
-		XCTAssertTrue(app.buttons["AppStrings.ExposureSubmission.primaryButton"].isEnabled)
+		XCTAssertTrue(app.buttons["AppStrings.ExposureSubmission.primaryButton"].waitForExistence(timeout: .medium))
```

Comment:
Hmm seems odd that someone tested for `isEnabled`. 
Was it a mistake or intention? 
Is the button disabled and we have to wait for it to become enabled? Loading or something?

Reply:
Hmm I think you found a good point: In this step it is required to check if the button is enabled or not. 
I will have a refactor for it. I think it is still nessecary to wait for existence before checking for enabled or not. I will have a try.